index.js

```diff
@@ -144,8 +144,9 @@ function MiamiVice() {
         let lines = [output.filter(noEmpty).join(' '), ...detailLines];
 
         if (err) {
-            trace = extract(err, 'trace', 'stack') || trace;
-            lines.push(...formatError(err, trace, msg));
+            const internalErr = Object.assign({}, err);
```

Comment:
Why not just duplicate the error object here? E.g. `err = Object.assign({}, err);` doesn't that ensure it's not modified for later use.